public/css/chat.css

```diff
@@ -5,7 +5,11 @@
   border: 1px solid #fff;
   bottom: 10px;
   right: 10px;
+<<<<<<< Updated upstream
```

Comment:
@andela-pijege  you have merge conflicts here, you should fix and rebase off latest develop please